test/controllers/GuidanceControllerSpec.scala

```diff
@@ -207,10 +249,16 @@ class GuidanceControllerSpec extends BaseSpec with GuiceOneAppPerSuite {
         .returns(Future.successful(Some(expectedPage)))
 
       private lazy val errorHandler = app.injector.instanceOf[config.ErrorHandler]
-      private lazy val view = app.injector.instanceOf[views.html.render_page]
-
-      lazy val target = new GuidanceController(errorHandler, view, mockGuidanceService, stubMessagesControllerComponents())
-      lazy val result: Future[Result] = target.getPage(path, None)(fakeRequest)
+      private lazy val view = app.injector.instanceOf[views.html.standard_page]
+      private lazy val questionView = app.injector.instanceOf[views.html.question_page]
+
+      lazy val target = new GuidanceController(errorHandler, 
```

Comment:
Intellij is reporting whitespace at the end of the lines.

Reply:
will clear